dev/MRTCore/SignConfig.xml

```diff
@@ -13,6 +13,10 @@
    <file src="__INPATHROOT__\x86\mrm.dll" signType="AuthenticodeFormer" /> 
    <file src="__INPATHROOT__\x86\Microsoft.ApplicationModel.Resources.dll"  signType="AuthenticodeFormer" /> 
  </job>
+ <job platform="ARM" certSubject="CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US" configuration="Release" dest="__INPATHROOT__\ARM\signed" jobname="MRT Core binary sign (ARM)" approvers="">
+   <file src="__INPATHROOT__\ARM\mrm.dll" signType="AuthenticodeFormer" /> 
```

Comment:
general question - did WinUI3 request ARM?  I was wondering about this a few weeks ago (whether you needed to build ARM).  The Project Reunion sample packaging built ARM.

Reply:
not yet. WinUI3 did request for ARM64. As supporting ARM is also part of priority and Project Reunion builds ARM, I figure MRT needs to build ARM too.